framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -850,7 +945,11 @@ private boolean skipNode(NodeWithAccessModifiers<?> node) {
           processField((FieldDeclaration) decl, (VariableElement) elt);
           break;
         case ENUM_CONSTANT:
-          processEnumConstant((EnumConstantDeclaration) decl, (VariableElement) elt);
+          if (decl instanceof FieldDeclaration) {
```

Comment:
Under what circumstances is the kind `ENUM_CONSTANT` but the run-time type is `FieldDeclaration`?